companies/sbahn-berlin.json

```
@@ -0,0 +1,24 @@
+{
+    "slug": "sbahn-berlin",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "S-Bahn Berlin GmbH",
+    "address": "S-Bahn Berlin GmbH\nDatenschutz\nElisabeth-Schwarzhaupt-Platz 1\n10115 Berlin\nDeutschland",
+    "phone": "+49 30 29743333",
+    "email": "Datenschutz.Regio@deutschebahn.com",
+    "sources": [
+        "https://sbahn.berlin/datenschutz/",
+        "https://sbahn.berlin/impressum/"
+    ],
+    "comments": [
+        "Die bestellte Datenschutzbeauftragte der S-Bahn Berlin GmbH ist Frau Chris Newiger.",
+        "Fragen oder Anregungen zur Datenverarbeitung bei der S-Bahn Berlin GmbH können auch an kundenbetreuung@s-bahn-berlin.de gestellt werden.",
+        "Die S-Bahn Berlin GmbH ist eine Tochtergesellschaft der DB Regio AG."
+    ],
+    "request-language": "de",
+    "quality": "verified"
+}
```

```suggestion
}

```
The files should end with a newline.